share/man/man7/compartmentalization.7

```diff
@@ -0,0 +1,113 @@
+.\" Copyright (c) 2022 Dapeng Gao <dapeng.gao@cl.cam.ac.uk>
+.\"
+.\" Redistribution and use in source and binary forms, with or without
+.\" modification, are permitted provided that the following conditions
+.\" are met:
+.\" 1. Redistributions of source code must retain the above copyright
+.\"    notice, this list of conditions and the following disclaimer.
+.\" 2. Redistributions in binary form must reproduce the above copyright
+.\"    notice, this list of conditions and the following disclaimer in the
+.\"    documentation and/or other materials provided with the distribution.
+.\"
+.\" THIS SOFTWARE IS PROVIDED BY THE AUTHORS AND CONTRIBUTORS ``AS IS'' AND
+.\" ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+.\" IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+.\" ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHORS OR CONTRIBUTORS BE LIABLE
+.\" FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+.\" DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+.\" OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+.\" HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+.\" LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+.\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+.\" SUCH DAMAGE.
+.\"
+.\" $FreeBSD$
+.\"
+.Dd $Mdocdate: October 26 2022$
+.Dt COMPARTMENTALIZATION 7
+.Os
+.Sh NAME
+.Nm compartmentalization ,
+.Nm c18n
+.Nd library-based software compartmentalization
+.Sh DESCRIPTION
+This file contains instructions for using the library-based compartmentalization
+(c18n) prototype.
+.Pp
+To launch a dynamically-linked pure-capability application with library-based
+compartmentalization, use the special runtime linker located at
+.Pa /libexec/ld-elf-c18n.so.1 .
+This can be done either explicitly, by invoking
+.Pp
+.Dl /libexec/ld-elf-c18n.so.1 Ar executable
+.Pp
+or implicitly, by setting the
+.Sy INTERP
+field of the program header of the target executable to the special linker's
+path.
+Said
+.Sy INTERP
+field can be inspected by invoking
+.Pp
+.Dl readelf -l Ar executable
+.Pp
+and can be set during static linking, typically by supplying the following
```

Comment:
I'd drop "static" here as such programs are dynamically linked.

Reply:
The 'static linking' here refers to when the executable is built from `.o` files with `ldd` (typically driven by `cc`).

Reply:
I worry this will lead people to think that static binaries are supported when that makes no sense. Maybe refer "the linking phase of compilation" or something if "linking" alone seems too ambiguous.